newEraCaps/flows/fulfillment/wmsFeed/headerItem.md

```diff
@@ -112,12 +114,14 @@ For Transfer Order, the value will be 4; for other types, the value will be 2. C
     * 2 = Air
     * 3 = Sagawa
   * This is mapped with the Carrier Party of the shipgroup.
+  * We expect it to be hardcoded "3".
 
 ## DLV PAYMENT (S-14)
 
 * **Description:** Payment Type/Gateway
 * **Position:** 1-1
 * **OMS Value:** 2
+* **Default Value:** 2
```

Comment:
Default value serves the same purpose as OMS value. Please remove duplicate here. We can stick to one pattern of listing a fixed OMS value.